lib/components/CheckBox/CheckBox.tsx

```diff
@@ -28,19 +29,20 @@ export const CheckBox: FunctionComponent<Props> = memo(
 		checked = false,
 		onClick = () => void 0,
 		onChange = () => void 0,
+		children = label,
```

Comment:
What is the point of a `label` prop? Legacy reasons? Just thinking - what if the developer pass down a `label` and `children`. ie. `<Checkbox label="hello">world</Checkbox>` - they won't get `label` at all. Im of the opinion here to not even have the `label` prop. Seeing as it's currently used, we could deprecate it.